Makefile

```diff
@@ -33,6 +34,8 @@ endif
 OS = $(shell go env GOOS)
 ARCH = $(shell go env GOARCH)
 
+BUNDLE_PACKAGE ?= ocs-osd-deployer
```

Comment:
@leelavg Why are we allowing the invoker to override the bundle package? Also if someone will run it with a different value the result will be unpredictable, as it will not create a new CSV with the new name, it will just create a patch for a resource that does not exist.

I would suggest just embedding the name `ocs-osd-deployer` in the two commands that use it.

Reply:
done.